ChangeLog.md

```diff
@@ -1,9 +1,5 @@
 # ChangeLog for shakespeare
 
-### 2.1.1
-
-* Add support for `TypeApplications` inside Shakespeare quasiquotes
-
```

Comment:
Is there a good reason why this changelog entry was removed? This looks like a mistake.

Reply:
On Hackage the version 2.1.1 is the last one and is already deprecated.
[https://hackage.haskell.org/package/shakespeare-2.1.1](https://hackage.haskell.org/package/shakespeare-2.1.1)

Reply:
Ok, but it shouldn't be removed even if it's deprecated. The deprecated version should continue to exist in the version history.